packages/web/src/pages/sign-up-page/pages/CreateEmailPage.tsx

```diff
@@ -55,11 +60,30 @@ export const CreateEmailPage = () => {
   const dispatch = useDispatch()
   const navigate = useNavigateToPage()
   const existingEmailValue = useSelector(getEmailField)
+  const [isWaitingForSocialLogin, setIsWaitingForSocialLogin] = useState(false)
+  const alreadyLinkedSocial = useSelector(getLinkedSocialOnFirstPage)
 
   const initialValues = {
     email: existingEmailValue.value ?? ''
   }
 
+  useEffect(() => {
```

Comment:
can this useEffect + state + handlers be part of a hook that we can use in CreateEmailPage and PickHandlePage?

Reply:
I think I can make this work though, I'll add it in